### PR TITLE
[Homework #6] - JS-Part 1 by Ivan Ishchenko

### PR DESCRIPTION
**Pull request by ViVse**

### CODE REVIEW

File: homeworks/ivan-ishchenko/06-js-basics-1/index.js [hunk 1]
```diff
@@ -0,0 +1,258 @@
+// task 1 http://www.codewars.com/kata/opposite-number
+const opposite = number => -number;
+
+// task 2 http://www.codewars.com/kata/basic-mathematical-operations
+const basicOp = (operation, value1, value2) =>
+  eval(value1 + operation + value2);
```
Comment (VitalikMelnyk) [inline homeworks/ivan-ishchenko/06-js-basics-1/index.js:6]: In future homeworks, avoid using `eval` function.
https://stackoverflow.com/questions/86513/why-is-using-the-javascript-eval-function-a-bad-idea

File: homeworks/ivan-ishchenko/06-js-basics-1/index.js [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+
+  items.splice(0, items.length, ...falseArr, ...trueArr);
+  return falseArr.length;
+};
+
+// task 8 http://www.codewars.com/kata/word-count
+// link is invalid
+
+// task 9 https://www.codewars.com/kata/find-the-odd-int/
+const findOdd = A => {
```
Comment (VitalikMelnyk) [inline homeworks/ivan-ishchenko/06-js-basics-1/index.js:78]: Please, choose more readable names.
E.g.: `A` is bad practice to use.

**Review by VitalikMelnyk — COMMENTED** [review]
Well Done @ViVse.
Looks great. Just review my notes and make small changes.
Thanks.

**Comment by ViVse** [comment]
@VitalikMelnyk fixed all issues.

**Review by VitalikMelnyk — APPROVED** [review]
Well Done @ViVse!
Approved.

**Pull request merged**